### DISCUSSION TITLE
Firefox is not affected after applying change of color (Ubuntu Firefox snap)

### DISCUSSION THREAD

**Discussion by Boulotaur2024** [GradienceTeam/Gradience#685] [Q&A]
I cannot seem to make that 'pure black color' go out of Firefox :

![Screenshot from 2023-01-01 16-51-49](https://user-images.githubusercontent.com/8013986/210176917-8bfd0d3a-6c17-40f7-b005-f976a9bdf5a3.png)

I want it to be 'Nordic' style just as Nautilus and the rest.
I don't know why it's not changing accordingly

My theme in Firefox is set to 'System Theme' :

![Screenshot from 2023-01-01 16-53-52](https://user-images.githubusercontent.com/8013986/210176938-a9cb1307-6829-4517-bb0d-4d4aa42b6ea0.png)

Not that it's a big deal but still...
Oh yeah I'm bored today..
Anyway thanks !

- Gradience 0.3.3 (Flatpak)
- Picked the 'Nordic Dark' user preset, applied it and then logged out

**Comment by daudix** [comment]
Do you have Firefox GNOME theme installed? This theme is required for work

https://github.com/rafaelmardojai/firefox-gnome-theme

**Reply by Boulotaur2024** [reply]
> Do you have Firefox GNOME theme installed? This theme is required for work
> 
> https://github.com/rafaelmardojai/firefox-gnome-theme

Ok I just installed it :

![Screenshot from 2023-01-01 18-20-03](https://user-images.githubusercontent.com/8013986/210179408-c7ed82ac-3c7d-4c38-980f-2ecb5966dfdf.png)

And then I re-applied the Nordic theme, logged out... but no success it's still black

EDIT : found a closed issue that does sound a lot like what I'm having now : 
https://github.com/GradienceTeam/Gradience/issues/635

Mmm... I don't have a 'CustomChrome.css' but a 'userChrome.css' here : 

![image](https://user-images.githubusercontent.com/8013986/210179679-0ad05a9f-ce29-483d-a8c2-7bb5318b6aa8.png)

Don't know if that can help... Permission access problem ?

My Firefox profile (about:support in FF) is at : 
/home/mitch/snap/firefox/common/.mozilla/firefox/ofkz7hp1.default

**Reply by daudix** [reply]
ah, you use snap, AFAIK we don't have support for snap, only native packages and flatpak

**Reply by Boulotaur2024** [reply]
> ah, you use snap, AFAIK we don't have support for snap, only native packages and flatpak

Ok Thanks
I will try to ditch the snap version of FF : 
https://www.omgubuntu.co.uk/2022/04/how-to-install-firefox-deb-apt-ubuntu-22-04
And let you know how it goes (probably will work fine)

FF in snap form is actually default on Ubuntu 22.10 (installed 2 days ago).
I didn't even have the choice

**Reply by daudix** [reply]
I will look if we can add snap support, but yeah, it's not nice to have snap by default

**Reply by tfuxu** [reply]
Tracked in issue #686. If I have time next week, I might work on fixing this.
